lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -994,6 +994,7 @@ data ApiNetworkParameters = ApiNetworkParameters
     , desiredPoolNumber :: !Word16
     , minimumUtxoValue :: !(Quantity "lovelace" Natural)
     , eras :: !ApiEraInfo
+    , maximumCollateralInputs :: !Word16
```

Comment:
Suggestion: make clear that this is a maximum input **_count_**, rather than some other maximum that applies to each individual input: 
```suggestion
    , maximumCollateralInputCount :: !Word16
```